mining/mining.go

```diff
@@ -542,10 +516,8 @@ func (g *BlkTmplGenerator) NewBlockTemplate(payToAddress bchutil.Address) (*Bloc
 	// However, since the total fees aren't known yet, use a dummy value for
 	// the coinbase fee which will be updated later.
 	txFees := make([]int64, 0, len(sourceTxns))
-	txSigOps := make([]int64, 0, len(sourceTxns))
 	txSigChecks := make([]int64, 0, len(sourceTxns))
-	txFees = append(txFees, -1) // Updated once known
-	txSigOps = append(txSigOps, coinbaseSigOps)
+	txFees = append(txFees, -1)          // Updated once known
```

Comment:
I know this has been like this for a while... but why would we ever want to append -1 to the fees?

Reply:
I would have rathered prepend the item when known... but whatever. =)